### PR TITLE
Remove Article Y, Add 2a4

### PR DESCRIPTION
**Pull request by Rouxles**
There are other potential wordings:

```
- 2a4) Comply with all reasonable safety measures set forth by local governments or the organization team and have it be announced clearly before the competition.
```

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -60,10 +60,11 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 
 ## <article-2><competitors><competitors> Article 2: Competitors
 
-- 2a) Any person may compete in a WCA competition if they (complemented by [Regulation Y3](regulations:regulation:Y3)):
+- 2a) Any person may compete in a WCA competition if they:
     - 2a1) Comply with WCA Regulations.
     - 2a2) Meet the competition requirements, which must be clearly announced before the competition.
     - 2a3) Are not suspended by the WCA Board.
+    - 2a4) Comply with all reasonable safety measures set forth by local governments and have it be announced clearly before the competition.
```
Comment (hanwu85) [inline wca-regulations.md]:
```suggestion
    - 2a4) Comply with all reasonable safety measures announced clearly before the competition which are set by local governments.
```

Comment (hanwu85) [reply wca-regulations.md]: It sounds more natural to me.

Comment (mirandapedro) [inline wca-regulations.md]:
```suggestion
    - 2a4) Comply with all reasonable safety measures deemed necessary by the WCA Delegate, which must be clearly announced before the competition.
```

New proposed wording after WCA Staff feedback.

**Review by Ivan-Brigidano — APPROVED** [review]

**Comment by c-goodyear** [comment]
Board approved. Can be merged on June 1.

**Pull request merged**